website/docs/docs/cloud/about-cloud/dbt-cloud-features.md

```diff
@@ -78,6 +78,12 @@ link="/docs/cloud/dbt-cloud-ide/develop-in-the-cloud"
     body="View the history of your runs and the model timing dashboard to help identify where improvements can be made to the scheduled jobs."
     link="/docs/deploy/run-visibility"
     icon="pencil-paper"/> 
+
+<Card
```

Comment:
is explorer avail for team/enterprise? if so, this should have an asterisk to signal that?